app/views/comments/new.html.erb

```diff
@@ -1,7 +1,47 @@
-<h1>add a comment</h1>
+<style>
+  h1 {
+    font-size: 2rem;
+    margin-bottom: 1rem;
+  }
+
+  .commentContainer {
+    background-color: #f5f5f5;
+    padding: 1rem;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.2);
+  }
+
+  .commenttext {
+    width: 100%;
+    height: 100px;
+    border-radius: 5px;
+    border: 1px solid #ccc;
+    padding: 0.5rem;
+    font-size: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  .formsubmit {
+    background-color: #007bff;
+    color: #fff;
+    border: none;
+    border-radius: 5px;
+    padding: 0.5rem 1rem;
+    font-size: 1rem;
+    cursor: pointer;
+    transition: background-color 0.3s ease;
+  }
+
+  .formsubmit:hover {
+    background-color: #0062cc;
+  }
+</style>
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job adding all the changes recommended by the previous reviewer. However I think it would be better if you move all your styling code in a specific styling file inside your `app/assets/stylesheets/` directory. This for best practices reasons.